jaeger-jaxrs2/src/test/java/com/uber/jaeger/propagation/SpanInfo.java

```diff
@@ -50,7 +52,12 @@ public SpanInfo(Span span) {
     traceID = stringContext.substring(0, stringContext.indexOf(':'));
     baggage = span.getBaggageItem(FilterIntegrationTest.BAGGAGE_KEY);
     sampled =
-        (stringContext.substring(stringContext.lastIndexOf(':') + 1).getBytes()[0] & (byte) 1) == 1;
+        (stringContext
+                        .substring(stringContext.lastIndexOf(':') + 1)
+                        .getBytes(StandardCharsets.UTF_8)[
+                    0]
+                & (byte) 1)
+            == 1;
```

Comment:
this is the problem with doing things like format as a mandatory step at check-in. makes me hate plugins like formatters and checkstyle. one way is to rewrite it into separate lines.